app/main.py

```diff
@@ -1,3 +1,16 @@
 def cache(func):
-    # Write your code here
-    pass
+    info = {}
+
+    def wrapper(*args):
+        nonlocal info
+        if args in info:
+            print('Getting from cache')
+            for name, inf in info.items():
```

Comment:
I think that you can just `return info[args]` - it will be the same as 
```python
for name, inf in info.items():
        if name == args:
                    return inf
```